api/src/main/resources/application.yml

```diff
@@ -5,5 +5,9 @@ server:
 mongodb:
   database: "mydb"
   collection: "message"
+  uri: "mongodb://34.78.196.126:27017"
 
-#logging.level.org.mongodb: debug
+management:
+  health:
+    diskspace:
+      enabled: false
```

Comment:
What is the reason of disabling it?

Reply:
I didn't reserve any diskspace. I dont need this, because I use app engine.
If diskspace checker runs, I will receive next warning: DiskSpaceHealthIndicator : Free disk space below threshold. Available: 0 byte.